### PR TITLE
{tools}[dummy/] htop v2.0.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz, Python 2.7.10
See https://gist.github.com/fed55b2177a5fd4697f8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5effc9f347985be94679 for a full test report.

File: easybuild/easyconfigs/h/htop/htop-2.0.0.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+homepage = 'http://hisham.hm/htop/'
+description = """An interactive process viewer for Unix"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://hisham.hm/htop/releases/%(version)s/']
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('ncurses', '6.0')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/h/htop/htop-2.0.0.eb]: it needs only this?

Comment (boegel) [inline easybuild/easyconfigs/h/htop/htop-2.0.0.eb]: according to the docs, yes

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**